instance/tests/models/test_openedx_monitoring_mixins.py

```diff
@@ -152,10 +153,11 @@ def test_update_monitoring_additional_email(self, mock_newrelic):  # pylint: dis
         'additional_monitoring_emails' to the existing monitors.
         """
         instance = OpenEdXInstanceFactory()
-        existing_monitor_ids = [str(uuid4()) for i in range(3)]
+        existing_monitor_ids = [str(uuid4()) for i in range(4)]
         existing_monitors = {}
-        existing_monitor_urls = [instance.url, instance.studio_url, instance.lms_preview_url]
-        for i in range(3):
+        existing_monitor_urls = [instance.url, instance.studio_url, instance.lms_preview_url,
+                                 instance.lms_extended_heartbeat_url]
```

Comment:
@lgp171188 nit picking here, we usually split lists like this:

```
existing_monitor_urls = [
    instance.url,
    instance.studio_url,
    instance.lms_preview_url,
    instance.lms_extended_heartbeat_url
]
```

Reply:
@rocioar, fixed it with 803676c